test/serializable_resource_test.rb

```diff
@@ -2,8 +2,13 @@
 
 module ActiveModel
   class SerializableResourceTest < Minitest::Test
+    class Profile < ::Model
```

Comment:
Maybe we shouldn't use the name model, as it's not actually part of AMS, and is part of sister gems?